bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/WebSocketConnection.java

```diff
@@ -54,16 +58,16 @@
 public class WebSocketConnection {
     private final Logger logger = LoggerFactory.getLogger(WebSocketConnection.class);
     private final Gson gson = new Gson();
-    WebSocketClient webSocketClient;
-    @Nullable
-    Session session;
-    @Nullable
-    Timer pingTimer;
-    @Nullable
-    Timer pongTimeoutTimer;
-    Listener listener;
-    boolean closed;
-    IWebSocketCommandHandler webSocketCommandHandler;
+    private final WebSocketClient webSocketClient;
+    private final IWebSocketCommandHandler webSocketCommandHandler;
+    private final Listener listener;
+
+    private @Nullable Session session;
+    private @Nullable Timer pingTimer;
+    private @Nullable Timer pongTimeoutTimer;
```

Comment:
Why are two separate Timer instances required? Can't you use just a single Timer for all scheduling tasks?